signbank/dictionary/templates/dictionary/gloss_detail.html

```diff
@@ -398,11 +398,11 @@ <h4 class="modal-title" id="glossvideo{{glossvideo.pk}}-ModalLabel">{% blocktran
                 </tr>
                 <tr>
                     <th>{% blocktrans %}Sign number:{% endblocktrans %}</th>
-                    <td id='sign-number'>{{gloss.id}}</td>
+                    <td id='filmbatch'>{{gloss.id}}</td>
```

Comment:
```suggestion
                    <td id='sign-number'>{{gloss.id}}</td>
```

This should stay as sign-number, right?

Reply:
Corrected this :smile: